client/stylesheets/_toCat.scss

```diff
@@ -4813,6 +4810,71 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 
+/*Options DropDown*/
+
+.optionsClick {
+	margin:100px auto;
+	width:200px;
+}
+
+.optionsClickv ul {
+	position:relative;
+	font-weight:900;
+}
+
+.optionsClick ul li {
+	position:relative;
+	list-style:none;
+	cursor:pointer;
+}
+
+.optionsClick ul li ul {
+	position:absolute;
+	left:0;
+	right:0;
+}
+
+.optionsClick ul .clicker {
+	position:relative;
+	background:#2284B5;
+	color:#FFF;
+}
+
+.optionsClick ul .clicker:hover,.click-nav ul .active {
+	background:#196F9A;
+}
+
+.optionsClick img {
+	position:absolute;
+	top:9px;
+	left:12px;
+}
+
+.optionsClick ul li a {
+	transition:background-color 0.2s ease-in-out;
+	-webkit-transition:background-color 0.2s ease-in-out;
+	-moz-transition: background-color 0.2s ease-in-out;
+	display:block;
+	padding:8px 10px 8px 40px;
+	background:#FFF;color:#333;
+	text-decoration:none;
+}
+
+.optionsClick ul li a:hover {
+	background:#F2F2F2;
```

Comment:
Color literals like `#F2F2F2` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F2F2F2` should be written as `#f2f2f2`<br>Line should be indented with spaces, not tabs<br>Colon after property should be followed by one space